<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 20pt &quot;Microsoft YaHei&quot;;
color:rgb(255, 255, 255);
padding:10px;</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="MainPageLayout" stretch="6,31,3,26,1,27,6">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="ocp1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="32,17,7,18,26">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="ocp5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ocp6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ocp7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="ocp2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="17,24,2,20,37">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="ocp8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ocp9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="17,9">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="Btn_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ocp11">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="ocp10">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="ocp3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="36,20,44">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="ocp12">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ocp_13">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="ocp4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>627</width>
     <height>662</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="MainDetailLayout" stretch="1,5,1,5,1,10">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Microsoft YaHei&quot;;
color:rgb(255, 255, 255);
padding:10px;</string>
      </property>
      <property name="text">
       <string>当日基本信息</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Microsoft YaHei&quot;;
color:rgb(255, 255, 255);
padding:10px;</string>
        </property>
        <property name="text">
         <string>温度</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Microsoft YaHei&quot;;
color:rgb(255, 255, 255);
padding:10px;</string>
        </property>
        <property name="text">
         <string>xx</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Microsoft YaHei&quot;;
color:rgb(255, 255, 255);
padding:10px;</string>
        </property>
        <property name="text">
         <string>风速</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Microsoft YaHei&quot;;
color:rgb(255, 255, 255);
padding:10px;</string>
        </property>
        <property name="text">
         <string>xx</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Microsoft YaHei&quot;;
color:rgb(255, 255, 255);
padding:10px;</string>
        </property>
        <property name="text">
         <string>日期</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Microsoft YaHei&quot;;
color:rgb(255, 255, 255);
padding:10px;</string>
        </property>
        <property name="text">
         <string>xx</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Microsoft YaHei&quot;;
color:rgb(255, 255, 255);
padding:10px;</string>
      </property>
      <property name="text">
       <string>电价信息</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1">
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Microsoft YaHei&quot;;
color:rgb(255, 255, 255);
padding:10px;</string>
        </property>
        <property name="text">
         <string>峰电价</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Microsoft YaHei&quot;;
color:rgb(255, 255, 255);
padding:10px;</string>
        </property>
        <property name="text">
         <string>0.677元/kWh</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Microsoft YaHei&quot;;
color:rgb(255, 255, 255);
padding:10px;</string>
        </property>
        <property name="text">
         <string>(6:00-22:00)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_13">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Microsoft YaHei&quot;;
color:rgb(255, 255, 255);
padding:10px;</string>
        </property>
        <property name="text">
         <string>谷电价</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_14">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Microsoft YaHei&quot;;
color:rgb(255, 255, 255);
padding:10px;</string>
        </property>
        <property name="text">
         <string>(22:00-6:00)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_15">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Microsoft YaHei&quot;;
color:rgb(255, 255, 255);
padding:10px;</string>
        </property>
        <property name="text">
         <string>0.337元/kWh</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Microsoft YaHei&quot;;
color:rgb(255, 255, 255);
padding:10px;</string>
      </property>
      <property name="text">
       <string>模块信息</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_16">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Microsoft YaHei&quot;;
color:rgb(255, 255, 255);
padding:10px;</string>
        </property>
        <property name="text">
         <string>这是一个用于占位的Textlabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
